src/components/ActionMenu/components/SecondaryAction/SecondaryAction.scss

```diff
@@ -27,4 +27,11 @@ $button-spacing: rem(12px);
       @include focus-ring($border-width: rem(0));
     }
   }
+
+  &.destructive {
```

Comment:
```suggestion
  &.Destructive {
```

Nit pick but I think they always use capitalized class names.

Reply:
Btw we could rely on a little trick to avoid adding a new class but it's maybe too unstable:

```scss
  &[class*='destructive'] {
      color: var(--p-text-critical);
    }
```

it's like a regex detecting that the class contains `destructive` hahaha

Reply:
@sylvhama I believe the convention is capitalized for class name and lower-case for modifiers.